_posts/patterns/2016-01-28-alerts.md

```diff
@@ -0,0 +1,108 @@
+---
+layout:         pattern
+title:          Alerts
+date:           2015-01-28 00:00:00
+
+categories:     patterns
+tags:
+- atomic
+- pattern
+- alerts
+
+slug:           alerts
+
+url_styles:     "patterns/_alerts.scss"
+
+description:    Examples of alerts in a semi-real world context.
+
+info:           Alerts provide information about...TBD
+---
+
+<h3 class="hd-6 example-set-hd">Error Alert with Title and Button</h3>
+<div class="example-set">
+    <div class="alert alert-error" role="alert" aria-labelledby="alert-title-error" tabindex="-1">
+        <span class="icon alert-icon icon-exclamation-triangle" aria-hidden="true"></span>
+
+        <div class="alert-message">
+            <h3 class="alert-title" id="alert-title-error">
+                We are unable to load this chart.
+            </h3>
+        </div>
+
+        <button type="button" class="alert-action btn-neutral btn">Read More</button>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Success Alert with Title and Message</h3>
+<div class="example-set">
+    <div class="alert alert-success" role="alert" aria-labelledby="alert-title-success" tabindex="-1">
+        <span class="icon alert-icon icon-check" aria-hidden="true"></span>
+
+        <div class="alert-message">
+            <h3 class="alert-title" id="alert-title-success">
+                Success
```

Comment:
Since this title is what' announced first, I suggest giving it more context. Something like "Your changes have been saved" would work. You could then use the copy below to add any relevant information that wouldn't be necessary to understanding the alert. Basically, if you can say what you need to say in a few works, let that be the title.